content/en/docs/reference/access-authn-authz/extensible-admission-controllers.md

```diff
@@ -628,6 +628,47 @@ So a webhook response to add that label would be:
 {{% /tab %}}
 {{< /tabs >}}
 
+Starting in v1.19, admission webhooks can optionally return warning messages that are returned to the requesting client.
```

Comment:
```suggestion
Starting in v1.19, admission webhooks can optionally return warning messages that are returned to the requesting client in HTTP Warning: headers, with a warning code of 299.
```
(is that correct? I haven't checked the actual code).

Reply:
correct